docs/front/datamodel/Override.md

```diff
@@ -0,0 +1,13 @@
+# Override
+\#v3  
+(not yet Implemented)  
+
+![Override](./assets/Override_v3.png)
+
+### Dependencies:  
+- [IFixture](./IFixture.md)
+- [ICueble](./ICueble.md)
+- [ICue](./ICue.md)
+
+### Description:
+The Override inherits from the [ICueble](./ICueble.md) interface. It overrides an ICueble as an old Value and sets another ICueble as value.
```

Comment:
Maybe change the second sentence to something like "It saves an old ICueble as oldValue and overrides it with another ICueble." (for clarity)